Automattic/wp-calypso
# Jetpack Scan: Reorganize sections in tab nav

#### Changes proposed in this Pull Request

Reorganize the _Scan_ section in wordpress.com and cloud.jetpack.com using a tab navigation, as part of the Jetpack.com alignment initiative.

#### Implementations notes

* Cards in the threat list are different than the ones in the mockups (see screenshots below). This should be a different PR.
* The `classname.is_jetpackcom` pattern doesn't seem pretty solid moving forwards. At least in the scan section we could try to avoid duplication of styles.

#### Testing instructions

1. Download the PR and run Calypso. Add the `flags=jetpack/features-section` query param.
2. Check that the layout and structure of the _Scan_ section matches the mockups (screenshots below)
3. Check that the Scanner and History functionalities still work
4. Check that the layout looks good on mobile as well
5. Check that the _Scan_ section is not available (via the menu or the routes directly) when you remove the query param
6. Run Jetpack cloud
7. Check on desktop and mobile that the layout and structure of the _Scan_ section are similar to Calypso (the _Jetpack Scan_ heading should not be present)
8. Check that the Scanner and History functionalities still work

#### Snapshots

**Scanner**
<img width="846" alt="Screen Shot 2020-06-12 at 9 49 38 AM" src="https://user-images.githubusercontent.com/1620183/84509898-62bd5780-ac92-11ea-9dc5-6e9000059f97.png">

**History**
<img width="846" alt="Screen Shot 2020-06-12 at 9 49 51 AM" src="https://user-images.githubusercontent.com/1620183/84509911-69e46580-ac92-11ea-80f4-ec843934b3a7.png">
